app/controllers/surveys_controller.rb

```diff
@@ -1,12 +1,15 @@
 class SurveysController < ApplicationController
   def index
-    @surveys = Survey.all.paginate(page: params[:page], per_page: 25)
-    @page = params[:page]
+    params[:page] = 1 if params[:page] == ''
+    @completed = params[:completed] == 'true'
```

Comment:
Here, `params[:completed]` is of type boolean, not string. So why couldn't you just say `@completed = params[:completed]`